src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -75,6 +83,58 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
     this.setState({ showCreatePipelineRunModal });
   };
 
+  handleChange = event => {
+    const inputValue = event.target.value;
+    this.setState({
+      currentFilterValue: inputValue
+    });
+  };
+
+  handleAddFilter = () => {
+    this.setState({
+      invalidFilter: false
+    });
+    const currentURL = this.props.match.url;
+    const { currentFilterValue = '' } = this.state;
+    if (!currentFilterValue.match('([a-z A-Z]+:[a-z A-Z]+,?)+')) {
```

Comment:
I want to be able to render an error on the page when an invalid filter is input so was using a component state variable and not in the store to achieve this as we have in many other components, I want to not allow the user to input that value at all. I'm not sure if I could achieve this with that method?

I will switch the logic and also add the number checking as you're right that wouldn't be covered by that pattern! 👍

Reply:
Sorry, meant to say state, not store. Not having it in the state would be much better but I see the difficulty, problem is this flip flopping of this value in the state is going to cause re-renders which isn't ideal. It'll do for now I guess, @AlanGreene Might have an idea when hes back.

Also be sure to include anything else that might appear in there (afaik it can be almost anything right ?)